ionic-angular-template/src/services/auth.service.ts

```diff
@@ -89,17 +87,17 @@ export class AuthService {
     await this.api.put(`${environment.apiUrl}/users/wallet`, {uid, wallet});
   }
 
-  async getUserDataIfAuthenticated() {
-    const loggedUser = JSON.parse(this.storageService.getItem('user'));
-    if (loggedUser) {
-      return (await firebase.firestore().collection('users').doc(`${loggedUser.uid}`).get()).data();
-    }
-    return null;
-  }
+  // async getUserDataIfAuthenticated() {
```

Comment:
Why this fragment is commented.

Reply:
I used it to initialize the subject but replaced it with "null".

Reply:
Let's delete the commented code!